src/lib/avatar/avatar.ts

```diff
@@ -25,6 +24,27 @@ declare global {
  * The custom element class behind the `<forge-avatar>` element.
```

Comment:
I've been removing these useless comments and using `@summary` and/or `@description` with something useful.